.github/workflows/publish.yml

```diff
@@ -36,3 +36,29 @@ jobs:
         run: cargo publish -p eppo
         env:
           CARGO_REGISTRY_TOKEN: ${{ secrets.CRATES_IO_TOKEN }}
+
+  publish_ruby:
```

Comment:
Inspired by https://github.com/Eppo-exp/ruby-sdk/blob/main/.github/workflows/publish.yml and updated versions